src/app/search/search-notebook-table/search-notebook-table.component.scss

```diff
@@ -0,0 +1,11 @@
+.mat-column-descriptorType {
+  max-width: 90px;
+}
+
+.mat-column-projectLinks {
+  max-width: 80px;
+}
+
+td.mat-cell {
+  border-bottom-style: none;
```

Comment:
Curious why this is needed for the notebook table, but not the workflows/tools table

Reply:
This is a verbatim copy of the workflows table styles.  Kimberley made some changes, possibly this related to this, to the release branch.  In this PR, I purposefully tried _not_ to do a lot of refactoring when adding the new notebook functionality, because it would cloud the core changes and make it harder to determine the source of any problems.  Recommend that after we merge the release branch back into develop, we take a look at how it's turned out, and schedule some more work if necessary...